platform/translation_process/translation_states.mdx

```diff
@@ -0,0 +1,71 @@
+---
+id: translation_states
+title: Understanding Translation States
+sidebar_label: Translation States
+description: Tolgee organizes translations into three states, each visually shown by a colored bar near the translated key.
+---
+
+import { ScreenshotWrapper } from '../shared/_ScreenshotWrapper';
+
+Translation management is important when you work in a big team and want to ensure that when you translate text, the quality meets your standards. Changing the status of each translation helps teams work together well. You might want to approve or flag a translation as ready. Tolgee has a feature to simplify the process of approval.
+
+## The Three Translation States
+
+Tolgee organizes translations into three states, each visually shown by a colored bar near the translated key:
+
+- **Gray**: Untranslated
+- **Yellow**: Translated but not reviewed
+- **Green**: Reviewed
+
+<ScreenshotWrapper
+  src="/img/docs/platform/translation-states-example.png"
+  alt="Quality assurance in translation three states"
+/>
+
+The color-coding feature helps managers and translators to check the status of a translation. This system makes it visual and easy to go through, especially if you are working with a large number of keys.
```

Comment:
I don't think the amount of keys is relevant. What is large? 100, 1000, 10000? It helps event when there is 50 keys...